pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -287,3 +295,86 @@ func getVolumeGroups(groups []string) ([]vgInfo, error) {
 
 	return vgs, nil
 }
+
+func createNS(r *ndctl.Region, uselimit uint) error {
```

Comment:
The logic in this function is complex enough that we should have a unit test which covers different scenarios.

Reply:
The particular code was authored by @okartau. I just moved the existing code from `pmem-ns-init.go` to here.

Reply:
Then tests were already missing before. Now would still be a good time to add them.

Reply:
Tests for this part of code is a bit complicated as the code depends on underlined PMEM hardware region properties(size, interleaves, max extent).  Current LVM unit tests are run on a pre-populated logical volume groups.

Reply:
The usual approach for cases like this is to introduce an interface and then test the code with a mock implementation of the interface.

Reply:
Okay. Please file an issue to track it.

Reply:
Filed #692